docs/make.jl

```diff
@@ -0,0 +1,17 @@
+using Documenter, SpinGlassTensors
+# makedocs(
+#     modules=[SpinGlassTensors],
+#     sitename="SpinGlassTensors.jl",
+#     format=Documenter.LaTeX()
+# )
+makedocs(
+    modules=[SpinGlassTensors],
+    sitename="SpinGlassTensors.jl",
+    format=Documenter.HTML(
+        prettyurls=get(ENV, "CI", nothing) == "true"
+    )
+)
+deploydocs(
+    repo="github.com/euro-hpc-pl/SpinGlassTensors.jl.git",
+    devbranch="sparse-doc"
```

Comment:
Should we remove `devbranch` parameter entirely before merging this PR?

Reply:
I think we should change it after merge. It had to be like, otherwise the docs would only build on `master`.